examples/workflows/script_annotations.py

```diff
@@ -0,0 +1,36 @@
+try:
+    from typing import Annotated  # type: ignore
+except ImportError:
+    from typing_extensions import Annotated  # type: ignore
+
+from hera.shared import global_config
+from hera.workflows import Artifact, Parameter, Workflow, script, Steps, ArtifactLoader
+
+global_config.experimental_features["script_annotations"] = True
+global_config.experimental_features["script_runner"] = True
+
+
+@script()
+def echo_all(
+    an_int: Annotated[int, Parameter(description="an_int parameter", default=1, enum=[1, 2, 3])],
+    a_bool: Annotated[bool, Parameter(description="a_bool parameter", default=True, enum=[True, False])],
+    a_string: Annotated[str, Parameter(description="a_string parameter", default="a", enum=["a", "b", "c"])],
```

Comment:
We can remove the enums here as we found they're only used in suspend templates.